src/include/sof/drivers/edma.h

```diff
@@ -95,4 +96,18 @@
 #define EDMA0_SAI_CHAN_RX_IRQ	349
 #define EDMA0_SAI_CHAN_TX_IRQ	349
 
+/* 8ulp chans */
+#define EDMA2_CHAN0    0
```

Comment:
8ULP specific defines should be in an 8ULP-only header. You can make your own platform edma.h header to have these defines.

Reply:
@paulstelian97 @TangleZ I don't think we want another header just for this. Maybe we can add 8ULP in the name of changes.

E.g EDMA2_8ULP_CHAN0 0